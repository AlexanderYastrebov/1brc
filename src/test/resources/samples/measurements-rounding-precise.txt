Rounding;33.6
Rounding;31.7
Rounding;21.9
Rounding;14.6
