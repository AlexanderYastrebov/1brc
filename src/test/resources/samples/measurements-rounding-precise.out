{Rounding=14.6/25.5/33.6}
